### PR TITLE
Delay "Add Goal" button activation till wss connects

### PR DESCRIPTION
**Pull request by aerosol**
Perhaps still better than indefinite spinner in high latency conditions...

reading material: https://3.basecamp.com/5308029/buckets/36789884/card_tables/cards/7954549712

### CODE REVIEW

File: lib/plausible_web/live/goal_settings/list.ex [hunk 1]
```diff
@@ -26,6 +27,9 @@ defmodule PlausibleWeb.Live.GoalSettings.List do
         >
           Add Goal
         </.button>
+        <.button :if={not connected?(@socket)} id="add-goal-button" mt?={false}>
```
Comment (zoldar) [inline lib/plausible_web/live/goal_settings/list.ex:30]: Shouldn't that be

```suggestion
        <.button :if={connected?(@socket)} id="add-goal-button" mt?={false}>
```

instead? 🤔

Comment (aerosol) [reply lib/plausible_web/live/goal_settings/list.ex:30]: :man_facepalming:

Comment (aerosol) [reply lib/plausible_web/live/goal_settings/list.ex]: lmao github broke - original code:

image

Comment (zoldar) [reply lib/plausible_web/live/goal_settings/list.ex:30]: I take that back, the condition was valid, I have not connected the dots that it's a button in _inactive_ state 🤡

**Review by zoldar — APPROVED** [review]

**Pull request closed without merging**